e2e/tests/conceptsets/conceptset-edit-rename.spec.ts

```diff
@@ -24,7 +23,7 @@ describe('Editing and rename Concept Set', () => {
    */
   test('Workspace OWNER can edit Concept Set', async () => {
 
-    const workspaceName = await findWorkspace(page, {create: true}).then(card => card.clickWorkspaceName());
```

Comment:
calling findWorkspace() with `{create: true}` and without a named workspace means *always create a new workspace* so effectively it's simply createWorkspace()